### PR TITLE
Enable /api/v1/targets endpoint on sampler

### PR DESCRIPTION
**Pull request by metalmatze**
As per request by @lilic

### CODE REVIEW

File: sampler.go [hunk 1]
```diff
@@ -112,21 +113,34 @@ func registerSampler(m map[string]setupFunc, app *kingpin.Application, name stri
...
 		if err := s.Run(context.TODO(), g, reloadCh); err != nil {
 			return nil, err
 		}
 
+		api := conprofapi.New(logger, reg, NopQuerier{}, reloadCh, 64*1024*1024,
+			func(ctx context.Context) conprofapi.TargetRetriever {
+				return scrapeManager
+			},
+		)
+		const apiPrefix = "/api/v1/"
```
Comment (lilic) [inline sampler.go:141]: Nitpick: Is this constant something you can define in the conprofapi package?

**Comment by brancz** [comment]
Amazing! lgtm :tada: :+1:

**Review by lilic — COMMENTED** [review]
Thanks so much, this is super helpful! 🎉

File: sampler.go (hunk 1 above)
Comment (metalmatze) [reply sampler.go:141]: Yes. I'll take care of that along with some other things to refactor in there.

Comment (lilic) [reply sampler.go:141]: Nice! :niceee:

**Pull request merged**